src/backend/commands/tablecmds.c

```diff
@@ -16058,9 +16058,11 @@ wack_pid_relname(AlterPartitionId 		 *pid,
 
 		par_prule = *ppar_prule;
 
-		*plrelname = par_prule->relname;
+		Assert(par_prule);
 
```

Comment:
remove this whitespace between assert and dereference

Reply:
Is that _really_ needed?

Reply:
Do you mean extra new line or am I missing anything?

Reply:
yes extra new line between assert and dereference

Reply:
Done

Reply:
Assertion should be at the place where a variable is initialized or assigned, postgres also uses this style